src/axios/endpoints.ts

```diff
@@ -16,7 +16,7 @@ const ENDPOINTS = {
   getMyAccommodations: (userId: string) => `/accommodations/${userId}/accommodations`,
   uploadAccomodationImage: (accommodationId: string) => `/accommodations/${accommodationId}/file`,
   deleteAccomodation: (accommodationId: string) => `/accommodations/${accommodationId}`,
-  getAllAccomodations: '/accommodations',
+  getAllAccomodations: (query: string) => `/accommodations${query}`,
```

Comment:
no need to pass the `query` as an argument, Axios is smart enough to put it right

Reply:
done